docs/concepts/copilot/copilot-ai-capabilities.md

```diff
@@ -0,0 +1,169 @@
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Fix with AI
+
+If your model has some errors, perhaps introduced upstream without your knowledge, Prophecy's Copilot will suggest fixes automatically.
+
+The Fix with AI option appears at any point where you see an error message.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/17q9stsz6n?videoFoam=true" title="Drag and Drop Model Graph Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Auto Documentation
+
+Understanding data assets is much easier with Data Copilot’s auto-documentation. Data Copilot delivers summary documentation suggestions for all Datasets, Pipelines, Models, and Orchestrations.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/9r86rl0lbz?videoFoam=true" title="Edit Code Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+How did a Dataset change? Data Copilot recommends a description of the change for every edit you make.
+
+<div class="wistia_responsive_padding" style={{padding:'62.5% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/rec6bcdwet?seo=false?videoFoam=true" title="Design a Pipeline Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_responsive_wrapper" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+How was a column computed? Data Copilot suggests a plain English description that explains data sources and how every column is generated and what it represents.
+
+This is a big time saver! You can edit the documentation suggestions and commit them to your repository.
+
+In addition to data sources and column summaries, Data Copilot also auto-documents your Project description, as well as your Git Commit Message and Release Note. And it can auto-label your Gems.
```

Comment:
this info seems to go with the first video in the documentation section?

Reply:
Done.

Reply:
Total Nit: 
This feels like a dangling idea. How about this: 
"Data Copilot auto-documents anywhere you need it - from the granular data sources and columns, through gem names, all the way to project description. Copilot even helps you write commit messages and release notes."

Reply:
Will address in https://github.com/SimpleDataLabsInc/prophecy-docs/pull/329 with some changes.